transformations/swap_characters/transformation.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class SwapCharactersPerturbation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
+
```

Comment:
Also, this won't be applicable for tagging if characters get exchanged with spaces. So, you need to only keep text classification, and text to text generation. And your changes should be mostly applicable to many other languages too - probably not to Chinese.

Reply:
you should change this to `languages="All"

Reply:
Done.